.github/workflows/backend.yml

```diff
@@ -4,10 +4,10 @@ on: [workflow_dispatch, push, pull_request]
 
 jobs:
   run:
-    uses: flarum/framework/.github/workflows/REUSABLE_backend.yml@1.x
+    uses: flarum/framework/.github/workflows/REUSABLE_backend.yml@2.x
     with:
-      enable_backend_testing: false
+      enable_backend_testing: true
       enable_phpstan: true
-      php_versions: '["7.4", "8.0", "8.1", "8.2"]'
+      php_versions: '["8.2", "8.3", "8.4"]'
```

Comment:
8.2, 8.3 and 8.4 are the defaults for Flarum 2 at present anyway, so I'd suggest removing this.

Will then also automatically start running with 8.5 when we add that to the reusable workflow